### PR TITLE
fix: taxonomies missing lang in sitemap

### PR DESCRIPTION
**Pull request by bemyak**
This PR fixes `sitemap.xml` having incorrect i18n taxonomy URLs.

---

## Sanity check:

* [X] Have you checked to ensure there aren't other open [Pull Requests](https://github.com/getzola/zola/pulls) for the same update/change?

## Code changes
* [X] Are you doing the PR on the `next` branch?

### CODE REVIEW

File: components/site/tests/site_i18n.rs [hunk 1]
```diff
@@ -169,6 +169,14 @@ fn can_build_multilingual_site() {
     assert!(file_contains!(public, "fr/tags/index.html", "bonjour"));
     assert!(!file_contains!(public, "fr/tags/index.html", "hello"));
 
+    // sitemap contains per-language taxonomies
+    assert!(file_contains!(public, "sitemap.xml", "https://example.com/tags/"));
+    assert!(file_contains!(public, "sitemap.xml", "https://example.com/tags/hello/"));
+    assert!(file_contains!(public, "sitemap.xml", "https://example.com/fr/tags/"));
+    assert!(file_contains!(public, "sitemap.xml", "https://example.com/fr/tags/bonjour/"));
```
Comment (bemyak) [inline components/site/tests/site_i18n.rs:176]: The current version puts `https://example.com/tags/bonjour/` here (notice missing `fr`), which results in lots of 404s in the crawlers.

File: components/site/src/sitemap.rs [hunk 2]
```diff
@@ -92,24 +92,19 @@ pub fn find_entries<'a>(
...
                 let number_pagers = (item.pages.len() as f64
                     / taxonomy.kind.paginate_by.unwrap() as f64)
                     .ceil() as isize;
                 for i in 1..=number_pagers {
                     let permalink = config.make_permalink(&format!(
-                        "{}/{}/{}/{}",
-                        name,
-                        item.slug,
+                        "{}{}/{}",
+                        item.path,
```
Comment (Keats) [inline components/site/src/sitemap.rs:107]: are we ensuring it ends with a trailing slash upstream already?

Comment (bemyak) [reply components/site/src/sitemap.rs:107]: Yes, `config.make_permalink` ensures it.
I've now added the slash nevertheless just for the consistency.

**Review by Keats — APPROVED** [review]

**Comment by Keats** [comment]
Thanks!

**Pull request merged**